lesson4/lib/news/services/News.js

```diff
@@ -0,0 +1,71 @@
+const rp = require('request-promise')
+const cheerio = require('cheerio')
+
+class News {
+  constructor() {
+    this.newsServiceUrl = ''
+    this.newsListUrn = ''
+    this.listItemSelector = ''
+    this.itemTitleSelector = ''
+    this.itemDateSelector = ''
+    this.itemBodySelector = ''
+  }
+
+  async getNewsList() {
+    let news = []
+
+    await rp(`${this.newsServiceUrl}${this.newsListUrn}`)
```

Comment:
Why is news array mutating here? It's more suitable to resolve it and then to catch it via await

Reply:
Yes. I'll have a closer look at it.